src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1543820426.inc.php

```diff
@@ -0,0 +1,56 @@
+<h1>Build #1543820426</h1>
+<h2>Date: 2018-12-03</h2>
+<div class="changelog">
+    - #135: Add cronjobs enabled flag
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      'name' => 'cronjobs_enabled',
+      'translation' => 'Cronjobs erlaubt',
+      'cms_field_type_id' => TCMSLogChange::GetFieldType('CMSFIELD_BOOLEAN'),
+      'cms_tbl_field_tab' => '',
+      'isrequired' => '0',
+      'fieldclass' => '',
+      'modifier' => 'none',
+      'field_default_value' => '1',
+      'length_set' => '',
+      'fieldtype_config' => '',
+      'restrict_to_groups' => '0',
+      'field_width' => '0',
+      'position' => '2171',
+      '049_helptext' => 'Gibt an, ob Cronjobs ausgeführt werden sollen. Dies wird kurz vor der Ausführung eines jeden Cronjobs geprüft. Es kann also die Ausführung aller Cronjobs mittendrin unterbrochen werden.
+Dabei wird allerdings der gerade laufende nicht abgebrochen. #135',
```

Comment:
```suggestion
```

Reply:
I removed the issue number on purpose in the proposal. The number is of no use for end users and will lead to confusion.

Reply:
ok (2x)